ui/StatusQ/src/StatusQ/Core/Utils/Utils.qml

```diff
@@ -51,15 +51,17 @@ QtObject {
 
     function findAssetByChainAndSymbol(chainIdToFind, assets, symbolToFind) {
         for(var i=0; i<assets.rowCount(); i++) {
-            const symbol = assets.rowData(i, "symbol")
-            if (symbol.toLowerCase() === symbolToFind.toLowerCase() && assets.hasChain(i, parseInt(chainIdToFind))) {
+            const symbol = ModelUtils.get(assets, i, "symbol")
+            const chainId = ModelUtils.get(assets, i, "chainId")
+            if (symbol.toLowerCase() === symbolToFind.toLowerCase() && chainIdToFind === chainId) {
```

Comment:
probably there is no chainId in assets view. Assets have `balances` submodel with `chainId` role. My presumption is that `hasChain` was looking into that nested model. I would keep it as it was and create separate ticket to wait for @Khushboo-dev-cpp is back or sb else from wallet can take a look.

Reply:
It's just for the assets selector used in the Browser section afaics

Reply:
In case of further problems with this code -`hasChain` has been removed here: https://github.com/status-im/status-desktop/commit/ff951f624aae21e80a78c8b95bd63574aca1de86

Reply:
OK, fixed using looking into the submodel with `getByKey(balances, "chainId", chainIdToFind)`